Classes/Domain/Model/Company.php

```diff
@@ -240,17 +188,11 @@ public function removeLogo(FileReference $logo): void
         $this->logo->detach($logo);
     }
 
-    /**
-     * @return array|FileReference[]
```

Comment:
For better type hinting in IDEs I prefer to keep array|FileReference[]

Reply:
Done